app/components/Swimlane/header.jsx

```diff
@@ -59,6 +63,7 @@ const SwimlaneHeader = React.createClass({
   render() {
     return (
       <header className="swimlane__header">
+        <button className="swimlane__header__button" onClick={ this.handleSwimlaneToggle } >Toggle Lane</button>
```

Comment:
Bit of a nitpick but there is an additional space between the `}` and the `>` which could be removed.